src/sentry/static/sentry/app/views/projectReleases/releaseLanding.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import {t} from 'app/locale';
+import {
+  BashCard,
+  Issues,
+  SuggestedAssignees,
+  Emails,
+  Contributors,
+} from 'sentry-dreamy-components';
+import ReleaseLandingCard from 'app/views/projectReleases/releaseLandingCard';
+import withApi from 'app/utils/withApi.jsx';
```

Comment:
Can we remove file ext?

Reply:
Ugh! My bad. I know you pointed this out before.